app/controllers/invoices_controller.rb

```diff
@@ -1,2 +1,9 @@
 class InvoicesController < ApplicationController
+  def update
+    invoice = Invoice.find(params[:id])
+    
+    invoice.update(status: params[:invoice][:status])
```

Comment:
I'm wondering if there is a strong params defined that could be used for this update.  Or is it easier to use this manual params argument since it's only the status being updated.